src/main/kotlin/org/jitsi/nlj/srtp/SrtpTransformer.kt

```diff
@@ -134,11 +134,10 @@ abstract class SrtcpTransformer(contextFactory: SrtpContextFactory) : AbstractSr
 /**
  * A transformer which decrypts SRTCP packets.
  */
-class SrtcpDecryptTransformer(contextFactory: SrtpContextFactory) : SrtcpTransformer(contextFactory) {
-    companion object {
-        val logger: Logger = Logger.getLogger(this::class.java)
-    }
-
+class SrtcpDecryptTransformer(
+    contextFactory: SrtpContextFactory,
+    parentLogger: Logger
+) : SrtcpTransformer(contextFactory, parentLogger) {
```

Comment:
I think we should derive a new logger for `SrtcpDecryptTransformer` here (and same for the 3 other implementations) in order to disambiguate where the logs are coming from, because right now they will all come from `SrtpTransformer` (also `SrtpTransformer` will need to **not** derive).